bahmni_config/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM httpd:2.4
+RUN apt update \
+&& apt install -y curl iproute2 iputils-ping less nano tree unzip
+RUN mkdir -p /var/www/
+COPY resources/httpd.conf /usr/local/apache2/conf/httpd.conf
```

Comment:
@Fabiencha , I believe we should expose the 8901 port here, for the people to know that the service runs on 8901.
This is applicable for Bahmni Apps as well (on port 8900)